data_prep/graph_dataset.py

```diff
@@ -57,24 +56,23 @@ def __init__(self, corpus, device):
         self._data.val_mask = val_mask
         self._data.test_mask = test_mask
 
-    # THIS FUNCTION IS NOT REALLY NEEDED
-    @property
-    def num_classes(self):
-        """
-        Return the number of unique classes in the dataset.
-        """
-        return self._num_classes
-
     def labels(self):
         """
-        Return the labels of datapoints.
+        Return the labels of data points.
 
         Returns:
             labels (Tensor): Document and word labels
         """
         return self._labels
 
-    def _generate_edges(self, tf_idf, tf_idf_words, pmi_scores, ntoi):
+    def as_dataloader(self):
+        """
+        Return this dataset as data loader from torch geometric.
+        """
+        return geom_data.DataLoader(self)
```

Comment:
Where do you import this?

Reply:
Good catch... while trying to run the models never came so far because of the memory :D